content/en/contribute/docs/style-guide.md

```diff
@@ -144,13 +144,18 @@ Connecting readers to related content in different pages is an important aspect
 1. **Inline links**: a portion of any narrative text can link to another page. This should done using the markdown link notation. 
    
    For example, the text `linking documents is a [foundational reason for the web existing in the first place](https://en.wikipedia.org/wiki/Hypertext)!` yields: "linking documents is a [foundational reason for the web existing in the first place](https://en.wikipedia.org/wiki/Hypertext)!"
-2. **See Also**: the `see-also` shortcode is available to connect to an important concept within the documentation site. The link will be more prominent to the reader by having a common prefix and shown on a separate line.
+   
+1. **See Also**: the `see-also` shortcode is available to connect to an important concept within the documentation site. The link will be more prominent to the reader by having a common prefix and shown on a separate line.
    
    For example, `{{</* see-also page="design/icons" */>}}` will show as seen here: {{< see-also page="design/icons" >}}
+   
+   You can also make the callout say "Read More" with the `prefix` tag: `{{</* see-also prefix="Read More" page="design/icons" */>}}`. This will show as seen here: {{< see-also page="design/icons" prefix="Read More" >}} 
+   
+   A custom title and anchor can be provided with this as well. For example, `{{</* see-also page="design/icons" title="Learn about the Icon Library" anchor="about-the-icon-library" */>}}`, will show as: {{< see-also page="design/icons" title="Learn about the Icon Library" anchor="about-the-icon-library" >}}
+   
+   Please use `see-also` when reference _related topics_ (e.g. [Workflows]({{< ref "apps/concepts/workflows" >}})) and use `read-more` when referencing the _same topic_ in more depth  (e.g. [Home Page]({{< ref "/" >}})) .  
```

Comment:
Fixed minor typo, and [we should avoid use of latin](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#latin-phrases). I've made a suggested edit, feel free to correct if you had intended it differently.

```suggestion
   Please use `see-also` when referencing _related topics_ , as seen in [Workflows]({{< ref "apps/concepts/workflows" >}}), and use `read-more` when referencing the _same topic_ in more depth, as in the [Home Page]({{< ref "/" >}}).  
```